plugin-build/src/main/kotlin/io/sentry/android/gradle/util/Versions.kt

```diff
@@ -15,11 +15,13 @@ internal object SentryVersions {
     internal val VERSION_PERFORMANCE = SemVer(4, 0, 0)
     internal val VERSION_OKHTTP = SemVer(5, 0, 0)
     internal val VERSION_FILE_IO = SemVer(5, 5, 0)
+    internal val VERSION_COMPOSE = SemVer(6, 7, 0)
 }
 
 internal object SentryModules {
     internal const val SENTRY_ANDROID_CORE = "sentry-android-core"
     internal const val SENTRY_ANDROID_OKHTTP = "sentry-android-okhttp"
+    internal const val SENTRY_ANDROID_COMPOSE = "sentry-android-compose"
```

Comment:
`h`: I think it should be `sentry-compose-android`, 'cause https://repo.maven.apache.org/maven2/io/sentry/sentry-compose-android/